### PR TITLE
Let's play with the new tool!

### PR DESCRIPTION
**Pull request by pattacini**
How great would it be if it only works!
<a href='#crh-start'></a><a href='#crh-data-%7B%22approved%22%3A%20%7B%22https%3A//github.com/pattacini%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3738070%3Fv%3D3%22%7D%2C%20%22https%3A//github.com/drdanz%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1100056%3Fv%3D3%22%7D%7D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/drdanz/yarp/pull/1%23issuecomment-208319438%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222016-04-11T12%3A37%3A29Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1100056%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/drdanz%22%7D%7D%2C%20%7B%22body%22%3A%20%22Yeah%20%3Ashipit%3A%20%22%2C%20%22created_at%22%3A%20%222016-04-11T12%3A38%3A25Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3738070%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/pattacini%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%20a034f75d83b35c0acc42ffe814788873278041ba%20README.md%202%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/drdanz/yarp/pull/1%23discussion_r59196045%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Also%20do%20not%20leave%20useless%20empty%20lines%20here%22%2C%20%22created_at%22%3A%20%222016-04-11T12%3A35%3A06Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1100056%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/drdanz%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222016-04-11T12%3A37%3A23Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1100056%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/drdanz%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20README.md%3AL1-6%22%7D%2C%20%22Pull%20a034f75d83b35c0acc42ffe814788873278041ba%20README.md%201%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/drdanz/yarp/pull/1%23discussion_r59195946%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22No%2C%20this%20is%20very%20wrong%22%2C%20%22created_at%22%3A%20%222016-04-11T12%3A34%3A02Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1100056%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/drdanz%22%7D%7D%2C%20%7B%22body%22%3A%20%22Oh%20yes%2C%20you%27re%20right%21%5Cr%5CnGoing%20to%20fix%20it%20up%21%22%2C%20%22created_at%22%3A%20%222016-04-11T12%3A35%3A10Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3738070%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/pattacini%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%20%22%2C%20%22created_at%22%3A%20%222016-04-11T12%3A35%3A47Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3738070%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/pattacini%22%7D%7D%2C%20%7B%22body%22%3A%20%22Blah%20%3A-1%3A%22%2C%20%22created_at%22%3A%20%222016-04-11T12%3A36%3A44Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1100056%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/drdanz%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222016-04-11T12%3A37%3A05Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1100056%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/drdanz%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20README.md%3AL1-6%22%7D%7D%2C%20%22processed%22%3A%20%5B%22https%3A//github.com/drdanz/yarp/pull/1%23issuecomment-208319438%22%2C%20%22https%3A//github.com/drdanz/yarp/pull/1%23issuecomment-208319672%22%2C%20%22https%3A//github.com/drdanz/yarp/pull/1%23discussion_r59195946%22%2C%20%22https%3A//github.com/drdanz/yarp/pull/1%23discussion_r59196045%22%2C%20%22https%3A//github.com/drdanz/yarp/pull/1%23discussion_r59196054%22%2C%20%22https%3A//github.com/drdanz/yarp/pull/1%23discussion_r59196134%22%2C%20%22https%3A//github.com/drdanz/yarp/pull/1%23discussion_r59196234%22%2C%20%22https%3A//github.com/drdanz/yarp/pull/1%23discussion_r59196273%22%2C%20%22https%3A//github.com/drdanz/yarp/pull/1%23discussion_r59196330%22%5D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>

<a href='https://github.com/pattacini'></a><a href='https://github.com/drdanz'></a>
- [x] <a href='#crh-comment-Pull a034f75d83b35c0acc42ffe814788873278041ba README.md 1'></a> &nbsp;<b><a href='https://github.com/drdanz/yarp/pull/1#discussion_r59195946'>File: README.md:L1-6</a></b>
- <a href='https://github.com/drdanz'></a> No, this is very wrong
- <a href='https://github.com/pattacini'></a> Oh yes, you're right!
  Going to fix it up!
- <a href='https://github.com/pattacini'></a> :+1:
- <a href='https://github.com/drdanz'></a> Blah :-1:
- [x] <a href='#crh-comment-Pull a034f75d83b35c0acc42ffe814788873278041ba README.md 2'></a> &nbsp;<b><a href='https://github.com/drdanz/yarp/pull/1#discussion_r59196045'>File: README.md:L1-6</a></b>
- <a href='https://github.com/drdanz'></a> Also do not leave useless empty lines here
- [x] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/drdanz/yarp/pull/1#issuecomment-208319438'>General Comment</a></b>
- <a href='https://github.com/drdanz'></a> :shipit:
- <a href='https://github.com/pattacini'></a> Yeah :shipit:

<a href='https://www.codereviewhub.com/drdanz/yarp/pull/1?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/drdanz/yarp/pull/1?approve=1'></a>&nbsp;<a href='https://www.codereviewhub.com/drdanz/yarp/pull/1?approve=0'></a>&nbsp;<a href='https://github.com/drdanz/yarp/pull/1'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by drdanz** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/drdanz/yarp/pull/1'>Refresh</a>.<a href='#crh-update-none'></a>

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+    This is my tiny edit!
```
Comment (drdanz) [inline README.md:1]: No, this is very wrong

File: README.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
+
```
Comment (drdanz) [inline README.md:2]: Also do not leave useless empty lines here

File: README.md (hunk 1 above)
Comment (pattacini) [inline README.md:1]: Oh yes, you're right!
Going to fix it up!

Comment (pattacini) [inline README.md:1]: :+1:

Comment (drdanz) [inline README.md:1]: Blah :-1:

Comment (drdanz) [inline README.md:1]: :+1:

File: README.md (hunk 2 above)
Comment (drdanz) [inline README.md:2]: :+1:

**Comment by drdanz** [comment]
:shipit:

**Comment by pattacini** [comment]
Yeah :shipit:

**Pull request closed without merging**